modules/rosa-installing.adoc

```diff
@@ -38,30 +38,40 @@ $ rosa
 .Example output
 [source,terminal]
 ----
-Command line tool for ROSA.
+Command line tool for Red Hat OpenShift Service on AWS.
+For further documentation visit https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws
```

Comment:
Doesn't this link to the documentation we are already viewing? Seems unnecessary.

Reply:
This isn't a link, it's part of example output for the tool.